apps/ledger-live-desktop/tests/page/accounts.page.ts

```diff
@@ -22,6 +31,30 @@ export class AccountsPage extends AppPage {
     await this.accountComponent(accountName).click();
   }
 
+  @step("Click show Account $0 tokens button")
+  async showParentAccountTokens(parentName: string) {
+    await this.accountComponent(parentName).scrollIntoViewIfNeeded();
+    await this.showTokensButton(parentName).click();
+  }
+
+  @step("Verify $0 children token accounts are not visible")
+  async verifyChildrenTokensAreNotVisible(parentName: string) {
+    await this.accountComponent(parentName).scrollIntoViewIfNeeded();
+    await expect(this.showTokensButton(parentName)).not.toBeVisible();
+  }
+
+  @step("Verify token visibility $1 having parent $0")
```

Comment:
I think second parameter "childCurrency: Currency" will be displayed as Object object in the reporting, so I guess we can remove it from step description